### ISSUE TITLE
Issue with electron cash 2.9.3

### ISSUE THREAD

**Issue by akap11** [fyookball/electrum#358]
I have BCC in my electron cash wallet. I try moving it over and get the error: the transaction was rejected by network rules. 16: mandatory-script-verify-flag-failed (Signature must use SIGHASH_FORKID)

I tried updating to the newest version but every time I try it crashes. I also tried to start a new wallet but don't have the seed.

Am I screwed?

**Comment by fyookball** [comment]
2fa wallet?

**Comment by akap11** [comment]
Yes it is. it has [2fa] on the top.
Im slightly new to this lol

**Comment by akap11** [comment]
is there anything I can do?

**Comment by KingParanha** [comment]
yes im having the exact same problem, i have also tried making a new wallet from the seed and disabling 2fa, but when i used that 2fa disabled wallet in 2.9,4 it just crashes? what can i do please ive been stuck like this all day
